pkdiagram/scene/event.py

```
@@ -75,166 +118,138 @@ def read(self, chunk, byId):
-        if self.parent:
-            self.parent.onEventProperty(prop)
-
-    def scene(self):
-        if self.parent:
-            if self.parent.isScene:
-                return self.parent
+        person = self.person()
+        if person:
+            person.onEventProperty(prop)
+
+    def scene(self) -> "Scene":
```

Ruff (0.13.3)

Undefined name `Scene`

(F821)